packages/components/app/styles/components/dropdown.scss

```diff
@@ -108,6 +109,34 @@ $hds-dropdown-toggle-border-width: 1px;
   }
 }
 
+.hds-dropdown-toggle-icon {
+  border-color: lightgray;
+  padding: 1px;
+}
+.hds-dropdown-toggle-icon__wrapper {
+  align-items: center;
+  border-radius: 3px; // $hds-dropdown-toggle-border-radius- 1px padding - 1px border
+  display: flex;
+  justify-content: center;
+  height: 32px;
+  width: 32px;
+
+  > * {
+    border-radius: inherit;
+  }
+}
+
+.hds-dropdown-toggle-icon--hasChevron {
+  margin-right: 0.25rem;
```

Comment:
this can be applied directly to the `hds-dropdown-toggle-icon__chevron` as `margin-left` without the need of this class (the icon/image is always present, while the chevron can be there or not, and we would apply the `margin-left` only to the chevron, only when it exists)